src/inverted-index.js

```diff
@@ -0,0 +1,140 @@
+const utils = require('./utils.js');
+
+/**
+ * A class for Creating and searching an inverted index
+ */
+class Index {
+
+  /**
+   * [constructor Creates Object variables to hold
+   * 1. [jsonDatabase] Uploaded files
+   * 2. [indexFile] the indexed filenames and their contents
+   * 3. [searchResult] the search results
+   */
+  constructor() {
+    this.jsonDatabase = {};
+    this.indexFile = {};
+    this.searchResult = {};
+    this.indexView = '';
+  }
+
+  /**
+   * [saveUploads creates a key and value object item that stores the uploaded file(s)]
+   * @param  {string} fileName [filename]
+   * @param  {object} jsonFile [content of uploaded json file]
+   * @return {boolean} [returns true on succesful addition of object to datatbase]
+   */
+  saveUploads(fileName, jsonFile) {
+    if (!utils.isValid(fileName, jsonFile)) {
```

Comment:
CRITICAL:
You may want to make this method name more explicit. `isValid` as a method name does not give me enough clues as to what it does.